### PR TITLE
Minor fix

### PR DESCRIPTION
**Pull request by navidcy**
@jbisits I notice triple ticks missing in the beginning of the example and I thought we fix it before we tag a new package version.

### CODE REVIEW

**Review by jbisits — COMMENTED** [review]
All looks good! I tried to follow the structure you use in `PassiveTracerFlows.jl` and `GeophysicalFlows.jl` for examples. I am happy for you to change this example in any way to better align with the way the examples run in `PassiveTracerFlows.jl` and `GeophysicalFlows.jl`

**Comment by jbisits** [comment]
Looking at the [example in the development docs](https://fourierflows.github.io/PassiveTracerFlowsDocumentation/dev/literated/turbulent_advection-diffusion/) it looks like I forgot some of the the `nothing` statements at the end of code blocks so there is some output. Should I add these in

**Review by jbisits — COMMENTED** [review]
Looks good!

**Comment by jbisits** [comment]
I was just trying to change a little bit of the wording in the example but was unable to push the commits from my computer upstream. Is this because this branch is associated with `PassiveTracerFlows.jl` and I do not have access? Nothing crazy just wanted to correct some of what is said when creating the time series for concentration and streamfunction.

**Comment by jbisits** [comment]
I will just re-review and write the changes there.

File: examples/turbulent_advection-diffusion.jl [hunk 1]
```diff
@@ -191,7 +194,7 @@ contour!(p, x, y, Array(ψ[1]'), levels = -0.15:-0.3:-1.5, lw=2, c=:grey, ls=:da
```
Comment (jbisits) [inline examples/turbulent_advection-diffusion.jl:194]: Suggest 

```julia
# Create time series for the concentration and streamfunction in the bottom layer, `layer = 2`.
file = jldopen(output.path)

iterations = parse.(Int, keys(file["snapshots/t"]))
t = [file["snapshots/t/$i"] for i ∈ iterations]

layer = 2

c = [file["snapshots/concentration/$i"][:, :, layer] for i ∈ iterations]
ψ = [file["snapshots/streamfunction/$i"][:, :, layer] for i ∈ iterations]
nothing # hide
```

to tidy up lines 160-170?

File: examples/turbulent_advection-diffusion.jl [hunk 2]
```diff
@@ -131,7 +134,7 @@ saveproblem(output)
 # We specify that we would like to save the concentration every `save_frequency` timesteps;
```
Comment (jbisits) [inline examples/turbulent_advection-diffusion.jl:134]: Add `nothing # hide` on line 85, 97 and 101?

File: examples/turbulent_advection-diffusion.jl [hunk 3]
```diff
@@ -148,7 +151,7 @@ while clock.step <= nsteps
   MultiLayerQG.updatevars!(params.MQGprob)
 end
```
Comment (jbisits) [inline examples/turbulent_advection-diffusion.jl:153]: Add `nothing # hide` at the end of this loop?

File: examples/turbulent_advection-diffusion.jl [hunk 4]
```diff
@@ -33,6 +34,7 @@ nothing # hide
       n = 128            # 2D resolution = n²
 stepper = "FilteredRK4"  # timestepper
      dt = 2.5e-3         # timestep
+nothing # hide
 
 # Physical parameters
```
Comment (jbisits) [inline examples/turbulent_advection-diffusion.jl:39]: Would this be clearer as a heading `### Physical parameters`?

**Review by jbisits — CHANGES_REQUESTED** [review]
Just a few things I saw that maybe were not as clear as they could be (from me!!)

**Comment by navidcy** [comment]
> I was just trying to change a little bit of the wording in the example but was unable to push the commits from my computer upstream. Is this because this branch is associated with `PassiveTracerFlows.jl` and I do not have access? Nothing crazy just wanted to correct some of what is said when creating the time series for concentration and streamfunction.

I gave you write access!

File: examples/turbulent_advection-diffusion.jl (hunk 1 above)
Comment (navidcy) [reply examples/turbulent_advection-diffusion.jl:194]: Yeap!

File: examples/turbulent_advection-diffusion.jl (hunk 3 above)
Comment (navidcy) [reply examples/turbulent_advection-diffusion.jl:153]: no need here

**Comment by jbisits** [comment]
> > I was just trying to change a little bit of the wording in the example but was unable to push the commits from my computer upstream. Is this because this branch is associated with `PassiveTracerFlows.jl` and I do not have access? Nothing crazy just wanted to correct some of what is said when creating the time series for concentration and streamfunction.
> 
> I gave you write access!

Ah sorry! I must have somehow got muddled with the branches/GitHub earlier.

File: examples/turbulent_advection-diffusion.jl (hunk 4 above)
Comment (navidcy) [reply examples/turbulent_advection-diffusion.jl]: Yeap

File: examples/turbulent_advection-diffusion.jl (hunk 2 above)
Comment (navidcy) [reply examples/turbulent_advection-diffusion.jl:134]: When the output is small then I like to show it. E.g., the output from constructing the `ADprob`..

**Comment by navidcy** [comment]
Thanks @jbisits. Re-review at your convenience.

**Review by jbisits — APPROVED** [review]
All looks good to me!

**Pull request merged**